torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16196,10 +16219,15 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.skip('Skipped!'), 'TestCommon', 'test_variant_consistency_eager'),
                DecorateInfo(unittest.skip('Skipped!'), 'TestJit', 'test_variant_consistency_jit'),
                DecorateInfo(unittest.skip('Skipped!'), 'TestMathBits', 'test_neg_view'),
-               # NotImplementedError: Could not run 'aten::tril_indices' with arguments from the 'Meta' backend.
-               DecorateInfo(unittest.expectedFailure, 'TestFakeTensorNonErroring', 'test_fake_autocast'),
-               DecorateInfo(unittest.expectedFailure, 'TestFakeTensorNonErroring', 'test_fake'),
-               DecorateInfo(unittest.expectedFailure, 'TestProxyTensorOpInfo', 'test_make_fx_fake_exhaustive'),
+               # test_dtypes will find this works for float dtypes, but it doesn't really
+               # make sense to use this with floats
+               DecorateInfo(unittest.skip('Skipped!'), 'TestCommon', 'test_dtypes'),
+               # for some small dtypes, the torch cpu implementation gives the wrong results
+               # when you run it for large input, so we skip those tests
+               DecorateInfo(unittest.skip('Skipped!'), 'TestDecomp', 'test_comprehensive',
+                            device_type="cpu", dtypes=(torch.uint8, torch.int8, torch.int16)),
+               DecorateInfo(unittest.skip('Skipped!'), 'TestDecomp', 'test_quick',
+                            device_type="cpu", dtypes=(torch.uint8, torch.int8, torch.int16)),
```

Comment:
Could you open an issue noting this?

Reply:
To be clear, it only gives the wrong results when you exceed the size of the dtype. I.e., if you ask for something with indices that would exceed 255 for uint8 like `torch.triu_indices(300,300, dtype=torch.uint8)`.
In those cases both current cuda implementation and the new ref implementation just give you the right result in int64 cast to the appropriate dtype, while the cpu implementation will give you 0 for the cases where the result would be over 255.

So, not clear if we should care about this. You could argue you should get an error when you exceed the dtype's max size.
Do you still think I should open an issue?

Reply:
FTR, discussed it Mario and I will make a separate PR so that CPU implementation errors when this issue arrises.